pkg/gtk4-layer-shell/build.zig

```diff
@@ -0,0 +1,105 @@
+const std = @import("std");
+
+const version: std.SemanticVersion = .{ .major = 1, .minor = 1, .patch = 0 };
```

Comment:
Should add a TODO that once we can import our `build.zig.zon` we should replace this with the version field from there.